.github/dependabot.yml

```diff
@@ -0,0 +1,9 @@
+version: 2
+updates:
+  - package-ecosystem: "gradle" 
+    directory: "/" 
+    target-branch: master
+    schedule:
+      interval: "daily"
+      time: "07:00"
+    open-pull-requests-limit: 5
```

Comment:
Is there any default devs group that can be added ? or maybe individual accounts?
Check dependabot.yaml from TS e.g.:
```
reviewers:
      - "Contrast-Security-Inc/teamserver"

```
or e.g.:

```
reviewers:
      - "cgperez"
      - "MissaelHRC"
      - "andrewboyd-contsec"
```

Reply:
Done